From d12486e15e144b6eb51ae5139d2c42b4f34e045a Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sat, 31 Dec 2016 08:43:01 +0000
Subject: [PATCH 2/2] panel: Move the default panel to the bottom

There's currently no real way around this: Brisk Menu doesn't really look
right at the top of the screen. It was built with the bottom in mind, and
provides a traditional Redmond-esque view of the world, so that's where
we'll have the panel now.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 panel/solus-live.layout | 14 +++++++-------
 panel/solus.layout      | 14 +++++++-------
 2 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/panel/solus-live.layout b/panel/solus-live.layout
index d37a5a9..f5d08f6 100644
--- a/panel/solus-live.layout
+++ b/panel/solus-live.layout
@@ -1,26 +1,26 @@
-[Toplevel top]
+[Toplevel bottom]
 expand=true
-orientation=top
+orientation=bottom
 size=24
 
 [Object brisk-menu]
 object-type=applet
 applet-iid=BriskMenuFactory::BriskMenu
-toplevel-id=top
+toplevel-id=bottom
 position=0
 locked=true
 
 [Object window-list]
 object-type=applet
 applet-iid=WnckletFactory::WindowListApplet
-toplevel-id=top
+toplevel-id=bottom
 position=10
 locked=true
 
 [Object notification-area]
 object-type=applet
 applet-iid=NotificationAreaAppletFactory::NotificationArea
-toplevel-id=top
+toplevel-id=bottom
 position=20
 panel-right-stick=true
 locked=true
@@ -28,7 +28,7 @@ locked=true
 [Object clock]
 object-type=applet
 applet-iid=ClockAppletFactory::ClockApplet
-toplevel-id=top
+toplevel-id=bottom
 position=10
 panel-right-stick=true
 locked=true
@@ -36,7 +36,7 @@ locked=true
 [Object show-desktop]
 object-type=applet
 applet-iid=WnckletFactory::ShowDesktopApplet
-toplevel-id=top
+toplevel-id=bottom
 position=0
 panel-right-stick=true
 locked=true
diff --git a/panel/solus.layout b/panel/solus.layout
index d37a5a9..f5d08f6 100644
--- a/panel/solus.layout
+++ b/panel/solus.layout
@@ -1,26 +1,26 @@
-[Toplevel top]
+[Toplevel bottom]
 expand=true
-orientation=top
+orientation=bottom
 size=24
 
 [Object brisk-menu]
 object-type=applet
 applet-iid=BriskMenuFactory::BriskMenu
-toplevel-id=top
+toplevel-id=bottom
 position=0
 locked=true
 
 [Object window-list]
 object-type=applet
 applet-iid=WnckletFactory::WindowListApplet
-toplevel-id=top
+toplevel-id=bottom
 position=10
 locked=true
 
 [Object notification-area]
 object-type=applet
 applet-iid=NotificationAreaAppletFactory::NotificationArea
-toplevel-id=top
+toplevel-id=bottom
 position=20
 panel-right-stick=true
 locked=true
@@ -28,7 +28,7 @@ locked=true
 [Object clock]
 object-type=applet
 applet-iid=ClockAppletFactory::ClockApplet
-toplevel-id=top
+toplevel-id=bottom
 position=10
 panel-right-stick=true
 locked=true
@@ -36,7 +36,7 @@ locked=true
 [Object show-desktop]
 object-type=applet
 applet-iid=WnckletFactory::ShowDesktopApplet
-toplevel-id=top
+toplevel-id=bottom
 position=0
 panel-right-stick=true
 locked=true
-- 
2.11.0

